spec/factories.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+  sequence :lat do |n|
+    "#{n}.0".to_f
+  end
+
+  sequence :lon do |n|
+    "#{n}.0".to_f
+  end
+
+  sequence :name do |n|
+    "name #{n}"
+  end
+
+  sequence :started_at do |n|
+    Time.zone.now + n.hours
```

Comment:
We've been using `Time.current` vs `Time.zone.now` as it using teh current zone. Is a bit shorted and then never forget to include `.zone.`

Also, another way to do this is: `n.hours.from_now`

Which will use the zone that you can initialize in your configs as

`Time.zone = 'UTC'` or East coast, etc.